keyboard_handler/src/keyboard_handler_unix_impl.cpp

```diff
@@ -0,0 +1,179 @@
+// Copyright 2021 Apex.AI, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef _WIN32
+#include <termios.h>
+#include <unistd.h>
+#include <string>
+#include <algorithm>
+#include <csignal>
+#include <iostream>
+#include <exception>
+#include "keyboard_handler/keyboard_handler_unix_impl.hpp"
+
+namespace
+{
+struct termios old_term_settings {};
+
+void quit(int sig)
+{
+  (void)sig;
```

Comment:
Unfortunately if I will try to use `void quit(int)` signature for function definition our `PC Lint` will throw an error and CI will not pass.

Reply:
Ok, that's fair.

Can you at least change it to not use a C-style cast? I think `static_cast<void>(sig);` should do the trick.

Reply:
@jhdcs I am sorry, but I am going to decline your request. \
And here are the rational for that:
- It's common approach in ROS code base to use C style cast to void for unused parameters or return value.
- It will looks "odd" `static_cast<void>(sig);` and IMO will cause even more confusions.
- Using `static_cast` versus C style cast make sense when need to be assure that types are compatible to be casted or when need at least to use casted value in future. In this particular case it doesn't matter. It's just workaround to avoid compiler warnings and errors.